### PR TITLE
12577 update correction title to Register Correction

### PR DESCRIPTION
**Pull request by lambert-alex**
*Issue #:* /bcgov/entity#12577

*Description of changes:*

- Correction header is now marked as Register correction
- App version = 3.4.7

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

**Review by lambert-alex — COMMENTED** [review]
When I merge PR 330 then I can fix the merge conflicts.

File: src/mixins/common-mixin.ts [hunk 1]
```diff
@@ -119,9 +119,9 @@ export default class CommonMixin extends Vue {
         return 'Record Conversion'
       case RouteNames.CORRECTION:
         if (this.isCorrectedIncorporationApplication) {
-          return 'Correction - Incorporation Application'
+          return 'Register Correction - Incorporation Application'
```
Comment (severinbeauvais) [inline src/mixins/common-mixin.ts:122]: I'm not certain what the requirement is, but I think it doesn't include the IA correction title. Please confirm with Linda/Dave and Yui. Thanks.

Comment (lambert-alex) [reply src/mixins/common-mixin.ts:122]: Okay I'll ask during stand up to confirm

**Comment by lewischenstudio** [comment]
I have a question about this ticket: [link](https://github.com/bcgov/entity/issues/12577). The following checkboxes are not displayed in the description:
- [ ] display only if client error (staff are able to enter the name that they have on the form)
- [ ] for staff error, verify that Filed By uses default user

Correct me if I'm wrong. In our current dev state, we are forcing the Edit UI to use `CorrectionErrorType == "STAFF"`. If that's the case, whether it's an client or staff error, the completing party will display the default user. Either way, the Edit UI doesn't display "Completing Party". Even if it does, it should display "Completing Party" with the input field filled by default user and the input field is disabled. 

If this is correct, then it will be nice to include those in this ticket.

**Comment by severinbeauvais** [comment]
> In our current dev state, we are forcing the Edit UI to use CorrectionErrorType == "STAFF". 

Subtle amplification: we're checking if type is CLIENT. If not (including if it's blank) then we assume STAFF.

File: src/mixins/common-mixin.ts (hunk 1 above)
Comment (severinbeauvais) [reply src/mixins/common-mixin.ts:122]: I heard in grooming today that it's "Register Correction" for all correction types, so this code can be simplified.

**Review by severinbeauvais — APPROVED** [review]

**Review by lewischenstudio — APPROVED** [review]

**Pull request merged**